### ISSUE TITLE
alis list: sort by user count

### ISSUE THREAD

**Issue by bukzor** [atheme/atheme#574]
Given that the LIST command is for when I don't know the channel name, I think alphabetical sorting isn't useful. Let's sort by the user count instead.

**Comment by bukzor** [comment]
@Zarthus prefers to add a -sort option rather than removing the alphabetical sort entirely. I'd still like to see user-count sorting be the default as I think that's more helpful in >90% of uses.

I'd also suggest that the highest count / first result is printed last, ensuring that it's visible to the user. Again @Zarthus suggests adding an option for this, but I believe the above behavior should be default.

**Comment by ant-222** [comment]
Yes, it is a good idea, especially in the form of a `sort_by` and `sort_ord` options to specity the field to sort by and the order (descending or ascenting). These can unified as:
`-sort` *field* `[`:(*asc*|*desc*)`]` 
Examples:
`-sort users_n:asc`
`-sort users_n:desc`
`-soft name`

**Comment by bukzor** [comment]
Closing — no longer pursuing this.
